### ISSUE TITLE
using share button for facebook prevent opening another window

### ISSUE THREAD

**Issue by JStone73** [nicolasbeauvais/vue-social-sharing#202]
Hi,

When in a overview i want to use a share button(see attachment) to share an article on facebook. However when clicking  the share button i want to prevent the underlying article detail to open. Is there a @click.prevent methode available for this plug-in?

best regards,
Jan

![share](https://user-images.githubusercontent.com/40293459/79633349-27584e00-8165-11ea-9c34-18de59f367dd.JPG)

**Comment by codesxt** [comment]
I have the same problem with an event card that I'm designing.

![Captura de pantalla de 2020-04-28 14-56-18](https://user-images.githubusercontent.com/5240873/80526272-80439580-8960-11ea-9f1d-ba0ac8904e9d.png)

There are two buttons that open modals with options. The first one opens a QR code and the second one opens a modal with the social share options.
The card is wrapped on a nuxt-link that sends the user to the event details page.

When I opened any button inside, it was triggering the nuxt-link so I added @click.stop to those buttons and they stopped triggering the link. But I can't do that for network elements. When I open any share option, it triggers the nuxt-link redirection.
